### PR TITLE
improve translation for user edit form

### PR DESCRIPTION
**Pull request by berteh**

### CODE REVIEW

File: budget/translations/fr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -107,7 +107,7 @@ msgstr "Nom"
 
 #: forms.py:155
 msgid "Weight"
-msgstr "Poids"
+msgstr "Coéfficient"
```
Comment (aavenel) [inline budget/translations/fr/LC_MESSAGES/messages.po]: Small typo here, should be "Coefficient"

**Comment by JocelynDelalande** [comment]
Good suggestion :-)

I user-tested quickly with two "innocent" people:

- "Coefficient" is better than "Poids"
- "Parts" seems even better

What do you think ?

**Comment by berteh** [comment]
I think "Parts" is even better, thanks, and happy new year!

**Comment by JocelynDelalande** [comment]
@berteh happy new year to you !

Please squash your commits :)

**Comment by berteh** [comment]
@JocelynDelalande from github document looks to me like this is something you can do on the merging in github directly... but I can't as I don't have write access to this repository: https://help.github.com/articles/about-pull-request-merges/#squash-and-merge-your-pull-request-commits

... but then maybe I didn't understand it correctly. if that's the case can you please tell me how to do it? sorry... not so familiar with git/github.

**Comment by JocelynDelalande** [comment]
> ... but then maybe I didn't understand it correctly. if that's the case can you please tell me how to do it? > sorry... not so familiar with git/github.

Oh you are right, it's a new-ish feature, I forgot about it. That said, squashing yourself your commit is still less work for the maintainer as s(he) do not have to write/decide on the commit message.

If you wan to do it yourself (anyway, always good to know how to do that), 

- `git rebase -i` your branch to squash the two commits (it's a rewrite of your git history)
- `git push --force` your modified branch to your topic-branch (`patch-1`).

(disclaimer : `push --force` is fine on a personal topic-branch like this one, but realy bad on a main/shared branch)

Thans a lot for your contribution, and all the best with git, it takes some efforts at the beggining but definitely worth it :-)



@bereth as a side note, I do not have merge rights on this repository.

**Comment by almet** [comment]
Hey! Thanks for your efforts :-) Do you need any other help to move this forward?

**Pull request closed without merging**